manifests/init.pp

```diff
@@ -1,10 +1,10 @@
 # Default strongSwan class.
 class strongswan(
   $package_name        = $strongswan::params::package,
-  $package_ensure      = 'installed',
+  $package_ensure      = $strongswan::params::package_ensure,
```

Comment:
Parameters that aren't OS specific are actually better off in the init.pp (where puppet-strings will be able to 'find' them)